resource-managers/kubernetes/core/pom.xml

```diff
@@ -47,6 +47,12 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
```

Comment:
This change is derived from [the previous PR](https://github.com/apache/spark/pull/16795/files)